README.md

```diff
@@ -2,7 +2,12 @@
 
 [![pdk-validate](https://github.com/ncsa/puppet-profile_additional_yumrepos/actions/workflows/pdk-validate.yml/badge.svg)](https://github.com/ncsa/puppet-profile_additional_yumrepos/actions/workflows/pdk-validate.yml)
  [![yamllint](https://github.com/ncsa/puppet-profile_additional_yumrepos/actions/workflows/yamllint.yml/badge.svg)](https://github.com/ncsa/puppet-profile_additional_yumrepos/actions/workflows/yamllint.yml)
+
  
+> :warning: This module is no longer recommended for configuring yum repos or yum proxy.
```

Comment:
I think we want to make this slightly more bold/obvious that it is deprecated. The warning alone doesn't quite stand out enough to me when I view the README.md on github.

I saw we had done something similar to the following in `ncsa/puppet-ssh`:

```suggestion
# DEPRECATED

> :warning: This module is no longer recommended for configuring yum repos or yum proxy.
```

Reply:
Ok, I moved the Deprecated message to the top and made it a top level heading.